test/orbit/import/personnel_test.exs

```diff
@@ -0,0 +1,64 @@
+defmodule Orbit.Import.PersonnelTest do
```

Comment:
Would it be possible to also have a test for `download()` that loads the data from `demo.csv`, so the loading and parsing and oban worker is tested?

If testing the oban worker is hard, you might still be able to test the rest of the loading process by refactoring some logic out of `Orbit.Import.PersonnelWorker.perform` and into a function in `Orbit.Import.Personnel`.

Reply:
Done!